docs/source/documentation/api_tokens_generation.rst

```diff
@@ -3,7 +3,24 @@ How-to use new API tokens for login
 
 This short guide explains how to manage API tokens in the web application, and use them in the Substra SDK.
 
-.. note::
+.. admonition:: Why generate API tokens?
+
+   The Substra SDK provides a way to log in using username and password (see `substra.Client <references/sdk.html#client>`_).
+   
+   It is safe, but should be used with caution:
+   
+   * It doesn't allow for a precise lifetime or separating concerns by creating one token per purpose.
+   
+   * It may surprise or limit you through its underlying automated session management.
```

Comment:
it might be worth expanding a bit on the automated session management. As a user who reads this, I know I am going to be surprised but I don't know how I can learn about it.

Reply:
I can expand on it, but I'm afraid we would then be documenting Substra internals rather than intended behavior. Just saying "it's magic, don't worry about it" allows us to change how it works later on.